Src/AutoFixtureUnitTest/DataAnnotations/StringLengthArgumentSupportTests.cs

```diff
@@ -24,15 +24,33 @@ public void FixtureCorrectlyCreatesShortText()
             // Teardown
         }
 
+        [Fact]
+        public void FixtureCorrectlyCreatesLongText()
+        {
+            // Fixture setup
+            var fixture = new Fixture();
+            // Exercise system
+            var actual = fixture.Create<ClassWithLengthConstrainedConstructorArgument>();
+            // Verify outcome
+            Assert.Equal(
+                ClassWithLengthConstrainedConstructorArgument.LongTextLength,
+                actual.LongText.Length);
+            // Teardown
+        }
+
         private class ClassWithLengthConstrainedConstructorArgument
         {
             public const int ShortTextMaximumLength = 3;
+            public const int LongTextLength = 50;
             public readonly string ShortText;
+            public readonly string LongText;
 
             public ClassWithLengthConstrainedConstructorArgument(
-                [StringLength(ShortTextMaximumLength)]string shortText)
+                [StringLength(ShortTextMaximumLength)]string shortText,
```

Comment:
In this particular case I don't think so. The class is used [only once](https://github.com/AutoFixture/AutoFixture/blob/master/Src/AutoFixtureUnitTest/DataAnnotations/StringLengthArgumentSupportTests.cs#L19), there are no any customizations applied. But I understand what you're talking about. Sometimes it might be hard to track if a test stopped working.

I'm fine with splitting it.